apps/web/components/ui/ProductCard/ProductCard.vue

```diff
@@ -29,6 +29,17 @@
       <p class="block py-2 font-normal typography-text-xs text-neutral-700 text-justify">
         {{ description }}
       </p>
+      <div class="flex items-center">
+        <span class="block pb-2 font-bold typography-text-sm" data-testid="product-card-vertical-price">
```

Comment:
<img width="268" alt="Bildschirmfoto 2023-08-24 um 22 47 59" src="https://github.com/plentymarkets/storefront-nuxt3-boilerplate/assets/30629157/44a8cd88-6207-4512-8f8a-589f5255ec97">

Reply:
price is there twice